qml/nextcloud/NewAccount.qml

```diff
@@ -200,11 +200,11 @@ Item {
                 }
             }
         }
-        request.open("POST", host + "/ocs/v1.php/person/check", true);
-        request.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
-        request.setRequestHeader("OCS-APIRequest", "true");
-        var body = "login=" + encodeURIComponent(username) + "&password=" + encodeURIComponent(password)
-        request.send(body);
+         var url = host + "/ocs/v2.php/cloud/user";
```

Comment:
Before, there was only one line with wrong indentation, now it's all of them :-)
Remove one space on every line, and they'll be OK. :-)